wp-web-push/lib/delapuente/wp-sw-concoct/class-wp-sw-concoct-router.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+class WP_SW_Concoct_Router {
+    private static $TRIGGER = '_wpswconcoct';
+
+    private static $instance;
+
+    // TODO: Perhaps it's better to separate init() from getting the singleton
+    public static function get_router() {
+        if (!self::$instance) {
+            self::$instance = new self();
+        }
+        return self::$instance;
+    }
+
+    private $routes;
+
+    public function add_route($url, callable $callback) {
```

Comment:
`callable` is only available from PHP 5.4, could it be a problem for us?

Looks like WordPress requires 5.2.6, but suggests 5.6: https://wordpress.org/about/requirements/